bin/git/git.js

```diff
@@ -78,6 +78,12 @@ export async function getVersion() {
   return result.replace('git version ', '');
 }
 
+export async function getSlug() {
+  const result = await execGitCommand(`git config --get remote.origin.url`);
+  const [, slug] = result.match(/([^/:]+\/[^/]+?)(\.git)?/) || [];
```

Comment:
Maybe a quick note of what this does? I don't read regex

Reply:
Fair point.